smartseq2_single_sample/check_reference/buildRef.wdl

```diff
@@ -0,0 +1,20 @@
+import "BuildStarReferenceBundle.wdl" as buildStarRef
+
+workflow buildStarRefs {
+  File ref_metadata_file
+  Array[Object] metadata = read_objects(ref_metadata_file)
```

Comment:
Unless it makes this more modular or easier to reuse, I find that this kind of input metadata abstraction (Array[Object]) makes the code more opaque. I would much rather have explicit inputs to the function in terms of individual arrays so I can see the inputs being scattered over.

Reply:
it is why I used object in wdl. it definitely made scatter easier.

Reply:
Would a comment about the format of the metadata object help clarify that @ambrosejcarr?

Reply:
Yea, I think it would. Mostly I want to be able to look at a workflow wdl and see:
1. what do I need to provide this file?
2. what do I get as output.